rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/class_annotations_target_entity.php.inc

```diff
@@ -18,8 +18,6 @@ final class ClassAnnotationsTargetEntity
 
 namespace Rector\Tests\Renaming\Rector\Name\RenameClassRector\Fixture;
 
-use Doctrine\ORM\Mapping as ORM;
```

Comment:
The alias is used in annotation.

Reply:
This is WIP, I'm working on those now

Reply:
Ok, I think the PostRector should be enabled only when defined via `$rectorConfig`, eg:

```php
$rectorConfig->removeUnusedUses();
```